apps/fz_http/lib/fz_http_web/controllers/json/rule_controller.ex

```diff
@@ -5,43 +5,30 @@ defmodule FzHttpWeb.JSON.RuleController do
 
   use FzHttpWeb, :controller
 
-  action_fallback FzHttpWeb.JSON.FallbackController
+  action_fallback(FzHttpWeb.JSON.FallbackController)
 
-  alias FzHttp.Rules
-
-  def index(conn, _params) do
-    # XXX: Add user-scoped rules
-    rules = Rules.list_rules()
-    render(conn, "index.json", rules: rules)
+  def index(conn, _) do
+    send_upgrade_notice(conn)
   end
 
-  def create(conn, %{"rule" => rule_params}) do
-    with {:ok, rule} <- Rules.create_rule(rule_params) do
-      conn
-      |> put_status(:created)
-      |> put_resp_header("location", ~p"/v1/rules/#{rule}")
-      |> render("show.json", rule: rule)
-    end
+  def create(conn, _) do
+    send_upgrade_notice(conn)
   end
 
-  def show(conn, %{"id" => id}) do
-    rule = Rules.get_rule!(id)
-    render(conn, "show.json", rule: rule)
+  def show(conn, _) do
+    send_upgrade_notice(conn)
   end
 
-  def update(conn, %{"id" => id, "rule" => rule_params}) do
-    rule = Rules.get_rule!(id)
-
-    with {:ok, rule} <- Rules.update_rule(rule, rule_params) do
-      render(conn, "show.json", rule: rule)
-    end
+  def delete(conn, _) do
+    send_upgrade_notice(conn)
   end
 
-  def delete(conn, %{"id" => id}) do
-    rule = Rules.get_rule!(id)
+  def update(conn, _) do
+    send_upgrade_notice(conn)
+  end
 
-    with {:ok, _rule} <- Rules.delete_rule(rule) do
-      send_resp(conn, :no_content, "")
-    end
+  defp send_upgrade_notice(conn) do
```

Comment:
While REST API is not released we probably can just drop this controller entirely in this PR and preventively rename it and it's params in the `master` branch. This way we won't have any hard-depreciation messages and breaking changes

Reply:
cc @jamilbk